.github/workflows/static-checks.yaml

```diff
@@ -60,7 +60,11 @@ jobs:
           test -z "$(cat $outfile)"
 
   golangci-lint:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+      fail-fast: false
+    runs-on: ${{ matrix.os }}
```

Comment:
I think on Windows you also need to install golang-ci via 
`choco install golang-ci /y`

Reply:
The workflow installation works, it's running on Windows that does not work for now but I suspect that once you have merged all your patch and this build and tests on Windows, it should work.

Reply:
The issue was that files under `tests` weren't building because of a dep, I added build ignore directives on those files.